pkg/instances/instances.go

```diff
@@ -24,9 +24,10 @@ import (
 )
 
 const (
-	daprEnabledAnnotation = "dapr.io/enabled"
-	daprIDAnnotation      = "dapr.io/app-id"
-	daprPortAnnotation    = "dapr.io/port"
+	daprEnabledAnnotation       = "dapr.io/enabled"
+	daprIDAnnotation            = "dapr.io/app-id"
+	daprPortAnnotation          = "dapr.io/port"
+	prometheusEnabledAnnotation = "prometheus.io/scrape"
```

Comment:
is there a specific reason that this is prometheus.io?

Reply:
Because we use prometheus forwarder, remember? So, this config is expected by that.

https://www.weave.works/docs/cloud/latest/tasks/monitor/configuration-k8s/

Reply:
Actually, wait. Hold on to this change. I think I found a mistake.